### PR TITLE
chore(release): bump to v6.1.0 alpha.3

### PR DESCRIPTION
**Pull request by sanni-t**
# Overview

Bump to v6.1.0 alpha.3

NOTE: Requires v1.0.2 h/s firmware bump in buildroot and a new buildroot release tag before tagging alpha release.

# Changelog

- Bumped package versions to alpha.3
- Updated user-facing api release notes
- lerna generated changelogs

# Review requests

- 6.1.0 Change-logs look reasonable? (there are unintended 6.0.1 changelogs in there because of 6.0.1 release syncing post-6.1.0 alpha2. But we will have to cleanup the logs before final release anyway so we will leave them as is for now)

# Risk assessment

Low.

### CODE REVIEW

File: api-client/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
+# [6.1.0-alpha.3](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.2...v6.1.0-alpha.3) (2022-08-19)
+
+
+
+## [6.0.1-alpha.1](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.1...v6.0.1-alpha.1) (2022-08-08)
+
+
+
+## [6.0.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.0.1-alpha.0) (2022-08-02)
+
```
Comment (bndo) [inline api-client/CHANGELOG.md:15]: will these get cleaned-up for release?

File: api/src/opentrons/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,37 @@
...
+
+
+
+## [6.0.1-alpha.1](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.1...v6.0.1-alpha.1) (2022-08-08)
+
+
+
+## [6.0.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.0.1-alpha.0) (2022-08-02)
+
+
```
Comment (bndo) [inline api/src/opentrons/CHANGELOG.md:28]: Same here, will these be cleaned prior to release?

File: app-shell/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,29 @@
...
+
+
+
+## [6.0.1-alpha.1](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.1...v6.0.1-alpha.1) (2022-08-08)
+
+
+
+## [6.0.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.0.1-alpha.0) (2022-08-02)
+
+
```
Comment (bndo) [inline app-shell/CHANGELOG.md:20]: Same here, will these be cleaned prior to release?

File: components/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,22 @@
...
+# [6.1.0-alpha.3](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.2...v6.1.0-alpha.3) (2022-08-19)
+
+
+
+## [6.0.1-alpha.1](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.1...v6.0.1-alpha.1) (2022-08-08)
+
+
+
+## [6.0.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.0.1-alpha.0) (2022-08-02)
+
```
Comment (bndo) [inline components/CHANGELOG.md:15]: Same here, will these be cleaned prior to release?

File: discovery-client/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,22 @@
...
+# [6.1.0-alpha.3](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.2...v6.1.0-alpha.3) (2022-08-19)
+
+
+
+## [6.0.1-alpha.1](https://github.com/Opentrons/opentrons/compare/v6.1.0-alpha.1...v6.0.1-alpha.1) (2022-08-08)
+
+
+
+## [6.0.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.0.1-alpha.0) (2022-08-02)
+
```
Comment (bndo) [inline discovery-client/CHANGELOG.md:15]: Same here, will these be cleaned prior to release?

**Review by bndo — APPROVED** [review]
Most of the `CHANGELOG.md`'s have extraneous info, but in the PR you mentioned they'd get cleaned up for release so assuming that's the part you had in mind, LGTM.

**Comment by sanni-t** [comment]
> Most of the `CHANGELOG.md`'s have extraneous info, but in the PR you mentioned they'd get cleaned up for release so assuming that's the part you had in mind, LGTM.

Yep, all the 6.0.1 mentions in the changelogs will be cleaned up for the release.

**Review by nusrat813 — APPROVED** [review]
LGTM

**Pull request merged**